docs/install/install-cave-beta.md

```diff
@@ -0,0 +1,165 @@
+# Install CAVE - BETA Version
+
+CAVE is the **C**ommon **A**WIPS **V**isualization **E**nvironment that is used for rendering and analyzing data for AWIPS.  The installer may require administrator priviledges to install and may require other system changes (environment variables, etc) as well.
+
+---
+
+## Latest CAVE Versions
+
+- [**Linux: 20.3.2-0.1**](#linux)
+- [**Windows:**](#windows) VM option available, but direct install not yet supported
+- **Mac:** not yet supported
+
+[**View release notes**](https://www.unidata.ucar.edu/blogs/news/tags/awips-release)
+
+!!! note "Version 20.* of CAVE is not compatible with Version 18.* EDEX and vice versa, Version 18.* of CAVE is not compatible with Version 20.* EDEX."
+
+---
+
+## General Requirements
+
+Regardless of what Operating System CAVE is running on, these general requirements are recommended in order for CAVE to perform optimally:
+
+- Local machine
+
+    !!! error "Running CAVE via X11 forwarding or ssh tunneling is **not** supported. Using a [VNC connection is the only remote option](../appendix/common-problems.md#remotely-connecting-to-cave), and may result in worse performance than running locally."
+  
+- Java 11
+- OpenGL 2.0 Compatible Devices
+- At least 4GB RAM
+- At least 2GB Disk Space for Caching
+- NVIDIA Graphics Card
+- [Latest NVIDIA Driver](http://www.nvidia.com/Download/index.aspx?lang=en-us)
+
+    !!! warning "While other graphics cards *may* work, NVIDIA Quadro graphics card is recommended for full visualization capability"
+
+---
+
+## Linux <i class="fa fa-linux"></i> 
+
+**Latest Version: 20.3.2-0.1**
+
+### System Requirements
+
+- 64 bit CentOS/Red Hat 7
+- Bash shell environment
+
+!!! note "While CentOS8 has reach End of Life as of Dec. 31, 2021, CentOS7 End of Life isn't until June 30, 2024."
+
+### Download and Installation Instructions
+
+1. Download the following installer: [**awips_install-v20.sh** <i class="fa fa-download"></i>](https://downloads.unidata.ucar.edu/awips2/20.3.2/linux/awips_install-v20.sh)
+2. In a terminal, go to the download directory 
+3. Make the installer an executable by running: `chmod 755 awips_install-v20.sh`
+4. Run the installer: `sudo ./awips_install-v20.sh --cave`
+     - This will install the application in `/awips2/cave/` and set the local cache to `~/caveData/`
+
+### Run CAVE
+
+To run CAVE either:
+
+- Use the terminal and type the command `cave`
+- Find the application in the Linux Desktop menu: Applications > Internet > AWIPS CAVE
+
+Additionally users can choose to run a [virtual machine (VM)](#linux-virtual-machine) on linux.
+
+---
+
+## Windows <i class="fa fa-windows"></i> 
+
+**Latest Version: 20.3.2-0.1**
+
+Currently the only installation available for Windows, is the [**Linux Virtual Machine**](#linux-virtual-machine).
+
+### Linux Virtual Machine
+
+Some CAVE functionality may be missing (ex: rendering RGB satellite images).
```

Comment:
In general CAVE will be missing functionality because it's a beta, so I think this sentence may be confusing... I would either take it out completely, or make it more clear, maybe with "Please note, running CAVE in a Virtual Machine does have reduced functionality than running CAVE directly on hardware (ex. rendering RGB satellite images)."